multiversx_sdk_core/transaction_parsers/transaction_on_network_wrapper.py

```diff
@@ -0,0 +1,115 @@
+from typing import List, Protocol, Sequence
+
+from multiversx_sdk_core.interfaces import IAddress
+
+
```

Comment:
can you extract the interfaces in a different file as you use them in transaction parser also? (e.g.ITransactionEvent, ITransactionOnNetwork)

Reply:
Indeed, extracted to `interfaces.py`.